front/app/component-library/hooks/useBreakpoint.ts

```diff
@@ -11,23 +11,23 @@ const mediaQueries = {
 type BreakpointType = keyof typeof mediaQueries;
 
 const useBreakpoint = (breakpointType: BreakpointType) => {
-  const [breakpoint, setBreakpoint] = useState(
+  const [matchesBreakpoint, setMatchesBreakpoint] = useState(
     window.matchMedia(mediaQueries[breakpointType]).matches
   );
 
   useEffect(() => {
-    const media = window.matchMedia(mediaQueries[breakpointType]);
-    if (media.matches !== breakpoint) {
```

Comment:
This condition is always false so it can be removed. Especially since it creates a dependency on `breakpoint` which causes this effect to be recreated and called needlessly.